### PR TITLE
Updated template, flavors and screenshots

### PR DESCRIPTION
**Pull request by prateektade**
Made the following changes -

1. Created `template.txt` in accordance with #1.
2. Built all flavors using `@rose-pine/build`.
3. Added screenshots for all flavors and updated instructions in readme.

### CODE REVIEW

**Review by mvllow — CHANGES_REQUESTED** [review]
Thank you for adding missing variants and polishing up this repo! I noted a few suggestions. Let me know if you need help with anything :)

File: readme.md [hunk 1]
```diff
@@ -13,23 +13,52 @@
...
+cat rosepine-moon.txt >> ~/.zshrc
+```
+
+**fish**
 
-# If you exclusively use Fish
+```sh
 cat rosepine.txt >> ~/.config/fish/config.fish
+cat rosepine-dawn.txt >> ~/.config/fish/config.fish
+cat rosepine-moon.txt >> ~/.config/fish/config.fish
```
[the remark this answers is not in the document]
Comment (prateektade) [reply readme.md:39]: I have tested it only on `bash` unfortunately. For `bash` and `fish` I just used the already available instructions, sorry for that.

File: readme.md [hunk 2]
```diff
@@ -13,23 +13,52 @@
...
 ```
 
 ## Gallery
 
-![Linux TTY with Rosé Pine](https://cdn.discordapp.com/attachments/767172954395639811/776229050804862976/IMG_20201111_153613.jpg)
+Fetch featured below is [NerdFetch](https://github.com/ThatOneCalculator/nerdfetch)
+
+**Rosé Pine**
+
+![Rosé Pine](https://raw.githubusercontent.com/prateektade/rose-pine-linux-tty/rose-pine-build-color-update/assets/Linux-TTY-Rosé-Pine.webp)
```
[the remark this answers is not in the document]
Comment (prateektade) [reply readme.md:48]: > As for the images themselves – we should avoid uploading them to the repo and instead use a permalink. I can assist with this if needed :)

Sure! Would you able to add the permalinks please?

File: readme.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (mvllow) [reply readme.md:39]: No worries, we can remove it for now if that's easiest and maybe someone can revisit it in a separate PR

[the remark this answers is not in the document]
Comment (prateektade) [reply readme.md:39]: We could use the instructions in the Usage section of the [rose-pine/fish](https://github.com/rose-pine/fish) repository I suppose.

File: readme.md [hunk 3]
```diff
@@ -13,23 +13,44 @@
...
+
+![Rosé Pine with Linux TTY](https://raw.githubusercontent.com/prateektade/rose-pine-linux-tty/rose-pine-build-color-update/assets/Linux-TTY-Rosé-Pine.webp)
+
+**Rosé Pine Dawn**
+
+![Rosé Pine Dawn with Linux TTY](https://raw.githubusercontent.com/prateektade/rose-pine-linux-tty/rose-pine-build-color-update/assets/Linux-TTY-Rosé-Pine-Dawn.webp)
+
+**Rosé Pine Moon**
+
+![Rosé Pine Moon with Linux TTY](https://raw.githubusercontent.com/prateektade/rose-pine-linux-tty/rose-pine-build-color-update/assets/Linux-TTY-Rosé-Pine-Moon.webp)
```
Comment (mvllow) [inline readme.md:48]:
```suggestion
![Rosé Pine Moon with Linux TTY](https://user-images.githubusercontent.com/1474821/204648344-7d5276c2-00af-44d9-8e17-11a60362f54d.jpg)
```

**Review by mvllow — APPROVED** [review]

**Comment by mvllow** [comment]
Thanks for making all the changes, much appreciated!

**Pull request merged**